### PR TITLE
Add prototype of sample for review

### PR DESCRIPTION
**Pull request by kellyhutchins**
@driskull  can you review this sample and provide feedback

### CODE REVIEW

**Comment by driskull** [comment]
Can we import calcite-web as an NPM package? https://esri.github.io/calcite-web/documentation/#npm

**Comment by driskull** [comment]
Can we have a `.ts` file for this example too?

**Comment by kellyhutchins** [comment]
@driskull yep but I want to get the JS one reviewed and tested before duplicating it in TS.

File: group-js-auth/app/Main.js [hunk 1]
```diff
@@ -0,0 +1,91 @@
...
+      var logInButton = document.getElementById("logIn");
+      var logOutButton = document.getElementById("logOut");
+      var logInPanel = document.getElementById("logInPanel");
+      var logOutPanel = document.getElementById("logOutPanel");
+      logOutButton.addEventListener("click", function () {
+        esriId.destroyCredentials();
+        logOutPanel.classList.add("hide");
+        logInPanel.classList.remove("hide");
+      });
+      watchUtils.watch(portal, "user", function () {
```
Comment (driskull) [inline group-js-auth/app/Main.js]: `watchUtils.watch(portal, "user", function (user) {`

```
logOutPanel.classList.toggle("hide", !user);
logInPanel.classList.toggle("hide",  user);
```

File: group-js-auth/app/Main.js [hunk 2]
```diff
@@ -0,0 +1,91 @@
...
+        // When user changes update data 
+        logOutPanel.classList.remove("hide");
+        logInPanel.classList.add("hide");
+        document.getElementById("username").innerHTML = portal.user.username;
```
Comment (driskull) [inline group-js-auth/app/Main.js]:
```
portal.user.username
portal.user.fullName
```
=>

```
user.username
user.fullName;
```

**Comment by driskull** [comment]
Maybe this sample needs a different name? It doesn't work with a group right? It just queries a user's items.

File: group-js-auth/app/Main.js (hunk 1 above)
Comment (kellyhutchins) [reply group-js-auth/app/Main.js]: Thanks I'll make these updates

**Comment by kellyhutchins** [comment]
@driskull  I should probably re-work it so it uses a group in both cases. I'll have to think about it.

**Comment by driskull** [comment]
Ok. if you want to use a group the ApplicationBase has a method to query items from a group.

**Comment by kellyhutchins** [comment]
yeah I just need to find a scenario that makes sense - maybe switch to groups and query for users groups instead of public groups once they sign-in

**Comment by driskull** [comment]
Is this PR still valid @kellyhutchins? If so, reopen.

**Pull request closed without merging**